UndertaleModTool/MainWindow.xaml.cs

```diff
@@ -3081,8 +3081,17 @@ private void SearchBox_TextChanged(object sender, TextChangedEventArgs e)
 
         public void UpdateObjectLabel(object obj)
         {
-            int foundIndex = obj is UndertaleNamedResource ? Data.IndexOf(obj as UndertaleNamedResource, false) : -1;
-            SetIDString(foundIndex == -1 ? "None" : (foundIndex == -2 ? "N/A" : Convert.ToString(foundIndex)));
+            int foundIndex = obj is UndertaleResource res
+                             ? Data.IndexOf(res, false)
+                             : -1;
+            string idString = foundIndex == -1
+                              ? "None"
+                              : (foundIndex == -2
+                                 ? "N/A"
+                                 : Convert.ToString(foundIndex)
+                                );
```

Comment:
I would personally use if elses here, instead of forcing ? : operators and make it less readable.